app/components/hideable-area/hideable-area.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react"
+import { Text } from "@rneui/themed"
+
+interface HideableAreaProps {
+  children: ReactNode
+  isContentVisible?: boolean
```

Comment:
why is this optional? there is probably no point of using the component without this value?